addon/components/code-base.js

```diff
@@ -0,0 +1,20 @@
+/* global Prism */
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  classNameBindings: ['languageClass'],
+  language: 'markup',
+
+  languageClass: computed('language', function() {
+    return 'language-' + this.get('language');
```

Comment:
are you avoiding using interpolation in case a consumer doesn't have support for es6?

```
return `language-${this.get('language')}`;
```

Reply:
Nope, this is a good idea 👍